NitroxServer/GameLogic/Players/PlayerData.cs

```diff
@@ -244,6 +278,19 @@ public PersistedPlayerData(string playerName, ushort playerId)
                 PlayerName = playerName;
                 PlayerId = playerId;
             }
+
+            /// <summary>
+            /// Overload for creating new PersistedPlayerData if a nitroxID is present
+            /// </summary>
+            /// <param name="playerName">the players name</param>
+            /// <param name="playerId">the players id</param>
+            /// <param name="nitroxId">the players NitroxID</param>
```

Comment:
IMO the documentation for these params don't add anything that isn't implied by their name already.